source/documentation/modules/math.html.md.erb

```diff
@@ -0,0 +1,494 @@
+---
+title: sass:math
+---
+
+<%= partial '../snippets/built-in-module-status' %>
+
+## Variables
+
+<% function 'math.$e' do %>
+  Equal to the value of the mathematical constant `e`.
```

Comment:
It's probably a good idea to link to the Wikipedia pages for most of these new members. Functions like `clamp()` doesn't need that since you can easily describe its behavior in prose, but it's good practice to assume that readers won't necessarily be coming from a mathematical background.